discovery-provider/alembic/versions/42d5afb85d42_create_audio_transactions_history.py

```diff
@@ -0,0 +1,68 @@
+"""create audio_transactions_history
+
+Revision ID: 42d5afb85d42
+Revises: a8752810936c
+Create Date: 2022-10-03 18:29:24.661763
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from src.utils.alembic_helpers import table_exists
+
+# revision identifiers, used by Alembic.
+revision = "42d5afb85d42"
+down_revision = "a8752810936c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # audio_transactions_history
+    if not table_exists("audio_transactions_history"):
+        op.create_table(
+            "audio_transactions_history",
+            sa.Column("user_bank", sa.String(), nullable=False),
+            sa.Column("signature", sa.String(), nullable=False),
+            sa.Column("transaction_type", sa.String(), nullable=False),
+            sa.Column("method", sa.String(), nullable=False),
+            sa.Column(
+                "created_at",
+                sa.DateTime(),
+                nullable=False,
+                server_default=sa.func.current_timestamp(),
+            ),
+            sa.Column(
+                "updated_at",
+                sa.DateTime(),
+                nullable=False,
+                server_default=sa.func.current_timestamp(),
+            ),
+            sa.Column("change", sa.BigInteger(), nullable=False),
+            sa.Column("balance", sa.BigInteger(), nullable=False),
```

Comment:
In the past we've used strings to represent numeric amounts (since WAudio amounts are padded out to ~10 digits). wdyt @rickyrombo ?

Reply:
Let me double check tipping endpoint, but if big int is big enough I'd say probably prefer a numeric type but convert to string in api?

Reply:
I think we're using bigint in `models/user_tip.py` and then convert to a string in the api.

Reply:
tips are bigint, balances elsewhere tend to be strings

Reply:
Solana amounts are unsigned BigInts, so I think we're fine with an unsigned BigInt. But postgres BigInt are signed, right? So maybe we need to do some offsetting if we don't want half the allowable size of an account

Reply:
https://solanacookbook.com/guides/get-program-accounts.html#filters